h/static/scripts/annotator/anchoring/range.coffee

```diff
@@ -144,7 +144,10 @@ class Range.BrowserRange
     # Look at the start
     if @startContainer.nodeType is Node.ELEMENT_NODE
       # We are dealing with element nodes
-      r.start = Util.getFirstTextNodeNotBefore @startContainer.childNodes[@startOffset]
+      if @startOffset < @startContainer.childNodes.length
```

Comment:
Can you add a bit of commentary as to what the problem this check is accounting for?